### PR TITLE
Feature/pp 68940

### PR DESCRIPTION
**Pull request by solancer**
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing](../blob/master/contributing.md) document?
* [x] Have you read and signed the automated Contributor's License Agreement?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by shreyanshp — APPROVED** [review]

File: src/lib/paypay-rest-sdk.ts [hunk 1]
```diff
@@ -15,28 +15,30 @@ export interface HttpsClientMessage {
 class PayPayRestSDK {
   private options: any = "";
   private productionMode: boolean = false;
+  private perfMode: boolean = false;
```
Comment (tilfin) [inline src/lib/paypay-rest-sdk.ts:18]: Why do you keep instance variables?

File: src/lib/paypay-rest-sdk.ts [hunk 2]
```diff
@@ -15,28 +15,30 @@ export interface HttpsClientMessage {
...
-  public configure = (clientConfig: { clientId: string; clientSecret: string; merchantId?: string; productionMode: boolean; }) => {
+  public configure = (clientConfig: { clientId: string; clientSecret: string; merchantId?: string; productionMode?: boolean; perfMode?: boolean; }) => {
     this.auth.setAuth(clientConfig.clientId, clientConfig.clientSecret, clientConfig.merchantId);
     if (clientConfig.productionMode) {
       this.productionMode = clientConfig.productionMode
     } else {
       this.productionMode = false;
     }
-    this.config = new Conf(this.productionMode);
+    if (clientConfig.perfMode) {
```
Comment (tilfin) [inline src/lib/paypay-rest-sdk.ts:38]: Optional boolean variables should be compared with `undefined` to determine if those are applied.

Comment (solancer) [reply src/lib/paypay-rest-sdk.ts:38]: @tilfin we are refactoring the whole SDK (repeating config files etc), we will be pushing that feature branch in a day or two please feel free to contribute on that draft PR.

File: src/lib/paypay-rest-sdk.ts (hunk 1 above)
Comment (solancer) [reply src/lib/paypay-rest-sdk.ts:18]: @tilfin we are refactoring the whole SDK (repeating config files etc), we will be pushing that feature branch in a day or two please feel free to contribute on that draft PR.

**Pull request merged**